person.rb

```diff
@@ -1,4 +1,5 @@
 require_relative('./nameable')
+require_relative('./rental')
 class Person < Nameable
   attr_reader :id
   attr_accessor :name, :age
```

Comment:
- [ ] Please make sure the `rentals` variable is readable and writable in the Person class to work as expected. 
  - You can consider the below code

```ruby
attr_accessor :name, :age, :rentals